pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/examples/ExampleViewModel.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.ui.examples
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import ch.srg.dataProvider.integrationlayer.data.remote.Media
+import ch.srg.dataProvider.integrationlayer.request.parameters.Bu
+import ch.srgssr.pillarbox.demo.shared.data.DemoItem
+import ch.srgssr.pillarbox.demo.shared.data.Playlist
+import ch.srgssr.pillarbox.demo.ui.integrationLayer.data.ILRepository
+import ch.srgssr.pillarbox.demo.ui.integrationLayer.di.IntegrationLayerModule
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.stateIn
+
+/**
+ * Example view model
+ *
+ * @param application Android Application to create [ILRepository]
+ */
+class ExampleViewModel(application: Application) : AndroidViewModel(application) {
+    private val repository: ILRepository = IntegrationLayerModule.createIlRepository(application)
+
+    /**
+     * Contents to display
+     */
+    val contents: StateFlow<List<Playlist>> = flow {
+        val listDrmContent = repository.getLatestMediaByShowUrn(SHOW_URN, 2).getOrDefault(emptyList())
+        val listTokenProtectedContent = repository.getTvLiveCenter(Bu.RTS, 2).getOrDefault(emptyList())
+        val playlist = createProtectedContentPlaylist(listDrmContent + listTokenProtectedContent)
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SyntheticAccessor: Access to `private` method `createProtectedContentPlaylist` of class `Companion` requires synthetic accessor